### ISSUE TITLE
Developer Contribution Workflow

### ISSUE THREAD

**Issue by TBN-MapleWheels** [PeasantFoundry/Mechtrauma#43]
In respect to discussion #39 , the new development process will be as follows.

- All development  will occur on the `dev` branch. PRs are to be tested separately and then merged into `dev`. Once features reach a point where it's satisfactory, the branch will be merged into `testing`. Any repo maintainers are only allowed to merge PRs aimed at the `dev` branch. The `testing` and `main` branches are only allowed to be written to by @PeasantFoundry .

- The `testing` branch will be used to test Release Candidates (RC) and no feature developments are to be done on this branch. Only balancing and bug fixes. Changes here will be merged into `dev`. Once the RC is ready, it will be merged into `main`

- `main` is only to contain ready-to-release versions of Mechtrauma. 

---

All major features must first be approved in an accompanying Issue or Discussion before merging the PRs into `dev`. 

Minor changes / tweaks / fixes only require the approval of @PeasantFoundry .
